lib/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Just as the reviewer mentioned to you to include the ```add_rental``` method, remember that for you to create a rental, you need to pass in the date, the details of the book and the person who is creating the rental. Therefore, the  ```add_rental``` method is used to establish a rental relationship between a book (self, which is an instance of the Book class) and a person (person, an instance of the Person class) with a specified date.